be/src/storage/tablet.cpp

```diff
@@ -614,7 +609,10 @@ const uint32_t Tablet::calc_cumulative_compaction_score() const {
             continue;
         }
 
-        score += rs_meta->get_compaction_score();
+        // non singleton delta means already compacted
+        if (rs_meta->is_singleton_delta()) {
```

Comment:
If one tablet has 100 cumulative deltas, the other has 1 cumulative delta. It will the score will be the same zero in the new algorithm. It's not right.

Reply:
cumulative delta will not compact in cumulative compaction